### DISCUSSION TITLE
How do I login to monkeytype api with postman?

### DISCUSSION THREAD

**Discussion by HyperDanisH** [monkeytypegame/monkeytype#4504] [Questions]
Hi,

I am working on the issue #4424. With every step I take I am being closer to solve the problem. Now, there is one issue which I think if someone could help me can help me a lot with this issue.

**How do I login to monkeytype api server with postman?**

Best,

**Comment by Miodec** [comment]
You have the server set to dev mode you can just pass in the uid in the body
![image](https://github.com/monkeytypegame/monkeytype/assets/13181393/1a34431c-c54d-49f2-acd7-5be8ed25b803)
